### PR TITLE
return mv cmd to build and test before doing mv

### PR DESCRIPTION
**Pull request by ajm01**

### CODE REVIEW

File: dev/devfile.yaml [hunk 1]
```diff
@@ -25,21 +25,18 @@ components:
...
       commandLine: if [ -e /projects/.disable-bld-cmd ];
                    then
                        echo "found the disable file" && echo "devBuild command will not run" && exit 0;
                    else
-                       echo "will run the devBuild command" && mvn -Dmaven.repo.local=/mvn/repository package
+                       echo "will run the devBuild command" && mkdir -p /projects/target/liberty 
+                                                            && if [ ! -d /projects/target/liberty/wlp ]; 
+                                                               then echo "...moving liberty"; mv /opt/ol/wlp /projects/target/liberty; 
+                                                               fi
+                                                            && mvn -Dmaven.repo.local=/mvn/repository package
```
Comment (scottkurz) [inline dev/devfile.yaml:39]: Small detail but maybe make the if/then indenting a bit more consistent:

```
      commandLine: if [ -e /projects/.disable-bld-cmd ]; then
                       echo "found the disable file" && echo "devBuild command will not run" && exit 0;
                   else
                       echo "will run the devBuild command" && mkdir -p /projects/target/liberty 
                                                            && if [ ! -d /projects/target/liberty/wlp ]; then 
                                                                   echo "...moving liberty"; mv /opt/ol/wlp /projects/target/liberty; 
                                                               fi
                                                            && mvn -Dmaven.repo.local=/mvn/repository package
                                                            && touch ./.disable-bld-cmd;
                   fi
```

File: dev/devfile.yaml [hunk 2]
```diff
@@ -25,21 +25,18 @@ components:
           targetPort: 9080
           protocol: http
 commands:
```
Comment (scottkurz) [inline dev/devfile.yaml:27]: Don't forget to bump up stack version.

**Review by scottkurz — CHANGES_REQUESTED** [review]
Small comments

File: dev/devfile.yaml (hunk 2 above)
Comment (ajm01) [reply dev/devfile.yaml:27]: bumped the version number

File: dev/devfile.yaml [hunk 3]
```diff
@@ -25,21 +25,20 @@ components:
...
-    exec:
-      component: devruntime
-      commandLine: echo "in the copy command"  && echo "...moving liberty"
-                                               && mkdir -p /projects/target/liberty
-                                               && mv -n /opt/ol/wlp /projects/target/liberty
-      workingDir: /
   - id: build
     exec:  
       component: devruntime 
       commandLine: if [ -e /projects/.disable-bld-cmd ];
```
Comment (scottkurz) [inline dev/devfile.yaml:31]: include in alignment.

File: dev/devfile.yaml [hunk 4]
```diff
@@ -25,21 +25,20 @@ components:
...
                    then
                        echo "found the disable file" && echo "devBuild command will not run" && exit 0;
                    else
-                       echo "will run the devBuild command" && mvn -Dmaven.repo.local=/mvn/repository package
+                       echo "will run the devBuild command" && mkdir -p /projects/target/liberty
+                                                            && if [ ! -d /projects/target/liberty/wlp ];
+                                                                 then echo "...moving liberty"; mv /opt/ol/wlp /projects/target/liberty; touch ./.liberty-mv;
+                                                               elif [[ -d /projects/target/liberty/wlp && ! -e /projects/.liberty-mv ]];
+                                                                 then echo "STACK WARNING - LIBERTY RUNTIME WAS LOADED FROM HOST";
```
Comment (scottkurz) [inline dev/devfile.yaml:39]: Warning msg would be confusing if we did a 2nd package after the 1st failed (but the mv had already been done).

Comment (scottkurz) [reply dev/devfile.yaml:39]: nm..you handled this..sorry

**Review by scottkurz — APPROVED** [review]
LGTM

**Pull request merged**